dpsctl/dpsctl.py

```diff
@@ -36,22 +37,26 @@
 
 """
 
+from __future__ import absolute_import, print_function
+
 import argparse
-import sys
 import os
 import socket
+import sys
+import threading
+import time
+
 import serial
-import protocol
-import uframe
-from uframe import *
-from uhej import *
 
+from . import protocol, uframe
```

Comment:
Would it be possible to make this Python 2 backwards compatible. Also, when testing with Python 3.6.1, I got the error ```ImportError: cannot import name 'protocol'```. Which is weird as protocol.py is in the same directory.